examples/external-data/src/mock-customer-service/service.ts

```diff
@@ -213,23 +255,28 @@ export async function initializeCustomerService(props: ServiceProps): Promise<Se
 	 *
 	 * ```json
 	 *	{
-	 *		containerUrl: string,
+	 *		documentId: string,
+	 *		tenantId: string,
 	 *		externalTaskListId: string
 	 *	}
 	 * ```
-	 *
-	 * Note: Implementers choice --can choose to break up containerUrl into multiple pieces
-	 * containing tenantId, documentId and socketStreamURL separately and send them as a json
-	 * object. The URL also contains all this information so for simplicity I use the URL here.
 	 */
 	expressApp.post("/register-session-url", (request, result) => {
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-		const containerUrl = request.body?.containerUrl as string;
+		const tenantId = request.body?.tenantId as string;
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		const documentId = request.body?.documentId as string;
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
 		const externalTaskListId = request.body?.externalTaskListId as string;
-		if (containerUrl === undefined) {
+		if (tenantId === undefined) {
+			const errorMessage =
```

Comment:
Let's rewrite the error messages a bit, they can be more like the one for externalTaskListId, or something like `"Required property 'tenantId' not provided in request body"`.